applications/nrf_desktop/src/modules/dvfs.c

```diff
@@ -66,16 +82,34 @@ enum dvfs_state {
 	LIST_FOR_DVFS_STATE(DVFS_STATE)
 };
 
-static const uint8_t dvfs_high_freq_bitmask = INITIALIZE_DVFS_FREQ_MASK(ACTIVE_FREQ_HIGH);
-static const uint8_t dvfs_medlow_freq_bitmask = INITIALIZE_DVFS_FREQ_MASK(ACTIVE_FREQ_MEDLOW);
+struct dvfs_frequency {
+	uint32_t freq;
+	uint8_t bitmask;
+};
+
+static const struct dvfs_frequency dvfs_high_freq = {
+	.freq = CONFIG_DESKTOP_DVFS_FREQ_HIGH,
+	.bitmask = INITIALIZE_DVFS_FREQ_MASK(ACTIVE_FREQ_HIGH),
+};
+
+static const struct dvfs_frequency dvfs_med_freq = {
+	.freq = CONFIG_DESKTOP_DVFS_FREQ_MED,
+	.bitmask = INITIALIZE_DVFS_FREQ_MASK(ACTIVE_FREQ_MEDLOW),
+};
+
+static const struct dvfs_frequency dvfs_low_freq = {
+	.freq = CONFIG_DESKTOP_DVFS_FREQ_LOW,
+	.bitmask = UINT8_MAX,
+};
 
 /* Binary mask tracking which states are requested. */
 static uint8_t dfvs_requests_state_bitmask;
 
-static enum dvfs_frequency_setting current_freq = DVFS_FREQ_HIGH;
+static uint32_t current_freq = dvfs_high_freq.freq;
 
-BUILD_ASSERT(sizeof(dvfs_high_freq_bitmask) == sizeof(dfvs_requests_state_bitmask));
-BUILD_ASSERT(sizeof(dvfs_medlow_freq_bitmask) == sizeof(dfvs_requests_state_bitmask));
+BUILD_ASSERT(sizeof(dvfs_high_freq.bitmask) == sizeof(dfvs_requests_state_bitmask));
```

Comment:
We could rely only on single assert here (as we compare bitmask size from the same struct type 3 times)